dev/ci/ci-checkov.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
```

Comment:
```suggestion

set -euxo pipefail

```

I think? This will help debug issues